### PR TITLE
Documentation (MKDOCS)

### PR DESCRIPTION
**Pull request by msa0311**
Initial creation of documentation, structure, content and styles using mkdocs.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: Publish docs via GitHub Pages
+on:
+  push:
+    branches:
+      - extensions-docs
```
Comment (jakolehm) [inline .github/workflows/main.yml]: Should this be `master`?

File: src/main/menu.ts [hunk 2]
```diff
@@ -193,6 +193,21 @@ export function buildMenu(windowManager: WindowManager) {
           navigate(whatsNewURL())
         },
       },
+<<<<<<< HEAD
```
Comment (jakolehm) [inline src/main/menu.ts]: This conflict needs to be resolved.

**Review by jakolehm — CHANGES_REQUESTED** [review]

File: src/main/menu.ts (hunk 2 above)
Comment (msa0311) [reply src/main/menu.ts]: done

File: .github/workflows/main.yml (hunk 1 above)
Comment (msa0311) [reply .github/workflows/main.yml]: yes, correct, just updated the pull request!

**Review by jakolehm — APPROVED** [review]

**Pull request merged**